cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js

```diff
@@ -86,21 +86,32 @@ function postMessage(message, callback) {
 function processEvent(event, callback) {
     console.log(event);
     const message = JSON.parse(event.Records[0].Sns.Message);
-    
-    var messageText;
+
+    let messageText;
     if (message.source == 'aws.config') {
         const alarmName = message.detail.configRuleName;
         const newState = message.detail.newEvaluationResult.complianceType;
         messageText = `${alarmName} state is now ${newState}`
+    } else if (message.source == 'aws.ssm') {
+        const eventName = message.detail.eventName;
+        const userName = message.detail.userIdentity.userName;
```

Comment:
On lines 100 and 101 you check if properties exist on detail; here you assume they do. Just want to make sure that this is intentional.

Reply:
Yes I believe these properties should always be there, especially since so far we are only capturing StartSession, ResumeSession and TerminateSession events.